config/config.go

```diff
@@ -80,7 +79,6 @@ func Parse() (Config, error) {
 
 	kingpin.Flag("address", "The address to listen on").Default(defaultAddress).StringVar(&config.Address)
 	kingpin.Flag("admin-group", "Tenant Admin Target Group").Required().StringVar(&config.AdminGroup)
-	kingpin.Flag("all-target-group", "View All Target Group").Required().StringVar(&config.AllTargetGroup)
```

Comment:
Apparently not a thing anymore.

https://gigantic.slack.com/archives/C7865GLSY/p1626876882006500